tensorflow/core/BUILD

```diff
@@ -1348,9 +1380,8 @@ tf_cuda_library(
         ":lib",
         ":lib_internal",
         ":protos_all_cc",
-        ":stream_executor",
         "//third_party/eigen3",
-    ],
+    ] + if_not_windows([":stream_executor"]),
```

Comment:
Instead of making this an (almost) no-op when building on Windows, could we modify `:core` so that it only pulls in `:gpu_runtime` on not-Windows?

There are a few targets that depend on `:gpu_runtime` in this file, but they all appear to depend on `:core_cpu`, so AFAICT this could be replaced with a dependency on `:core`.